src/Pages/Home/Home.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Header, Container, Link, H1, Paragraph, BigButton, Login} from './styled';
+
+
+
+export const Home = function () {

```

Comment:
please rewrite
`export const Home = function () {
    return (<div>Component</div>)
}` 
to 
`export const Home =() => (<div>Component</div>)`